api/RTCPeerConnection.json

```diff
@@ -4,26 +4,12 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection",
         "support": {
-          "chrome": [
-            {
-              "version_added": "56"
-            },
-            {
-              "prefix": "webkit",
-              "version_added": "51",
-              "version_removed": "56"
-            }
-          ],
-          "chrome_android": [
-            {
-              "version_added": "56"
-            },
-            {
-              "prefix": "webkit",
-              "version_added": "51",
-              "version_removed": "56"
-            }
-          ],
+          "chrome": {
+            "version_added": "23"
+          },
+          "chrome_android": {
+            "version_added": "25"
```

Comment:
This should be 23. Sorry if I forgot to include that in the last review.

Reply:
You did mention it, actually:

> It seems we don't have record of a Chrome for Android 23, so I set it to `25` instead (the closest, higher number available).

> I want to look into this, then, as a search for [Chrome for Android 23](https://www.google.com/search?q=site%3Achromereleases.googleblog.com%20chrome%2023) on the [Chrome Releases blog](https://chromereleases.googleblog.com) is pulling up no results, and [Wikipedia](https://en.wikipedia.org/wiki/Google_Chrome_version_history)* shows that there weren't any Android releases in between 18 and 25. Of course, if the MDN data is inaccurate and it did exist, we'll need to update the browser version info.
> 
> Do you know of any sources that confirm the existence of Chrome for Android 23?
> 
> _* Don't mind me citing Wikipedia as we've all been taught not to do..._

Reply:
25 then.